.env

```diff
@@ -1,8 +1,8 @@
-HOST=127.0.0.1
-PORT=8001
-
-DB_URL=127.0.0.1
-DB_PORT=27017
-FOOTAGE_COLLECTION_NAME=footage
-CLIPS_COLLECTION_NAME=clips
+HOST=127.0.0.1
+PORT=8001
+
+DB_URL=127.0.0.1
+DB_PORT=27017
+FOOTAGE_COLLECTION_NAME=footage
+CLIPS_COLLECTION_NAME=clips
 DB_NAME=waldo-footage
```

Comment:
A .env file shouldn't ever be commited, it can potentially contain secrects, thats why the example file exists.